### PR TITLE
Script

### PR DESCRIPTION
**Pull request by AndreyKorol**

### CODE REVIEW

File: script.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env ruby
+
+require_relative 'parser'
+require 'benchmark'
```
Comment (AndreyKorol) [inline script.rb]: Delete redundant require

Comment (xxkvl) [reply script.rb]: why do you talk to youself? man, is everythig okay? LGTM?

Comment (AndreyKorol) [reply script.rb]: Haha, hello Dmitriy! :) Yep, i like make reviews. I can miss some defects, so self review help me.

Comment (AndreyKorol) [reply script.rb]: Maybe you want to do some review of my code? I would appreciate.

**Pull request merged**